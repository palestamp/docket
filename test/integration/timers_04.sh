rm ~/.docket_timer_test
set -o verbose
docket timer new work --abstract
docket timer new work_child --parent work
docket timer new work_child2 --parent work
docket timer new work_child3 --parent work

docket timer start work_child
docket timer stat --status-only
docket timer start work_child2
docket timer stat --status-only
docket timer start work_child3
docket timer stat --status-only

docket timer stop work_child2
docket timer stat --status-only
docket timer stop work_child
docket timer stat --status-only
docket timer stop work_child3
docket timer stat --status-only
