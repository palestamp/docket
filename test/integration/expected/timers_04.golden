docket timer new work --abstract
docket timer new work_child --parent work
docket timer new work_child2 --parent work
docket timer new work_child3 --parent work

docket timer start work_child
docket timer stat --status-only
*               running   
work            running   
work_child      running   
work_child2     stopped   
work_child3     stopped   
docket timer start work_child2
docket timer stat --status-only
*               running   
work            running   
work_child      running   
work_child2     running   
work_child3     stopped   
docket timer start work_child3
docket timer stat --status-only
*               running   
work            running   
work_child      running   
work_child2     running   
work_child3     running   

docket timer stop work_child2
docket timer stat --status-only
*               running   
work            running   
work_child      running   
work_child2     stopped   
work_child3     running   
docket timer stop work_child
docket timer stat --status-only
*               running   
work            running   
work_child      stopped   
work_child2     stopped   
work_child3     running   
docket timer stop work_child3
docket timer stat --status-only
*               stopped   
work            stopped   
work_child      stopped   
work_child2     stopped   
work_child3     stopped   
